### PR TITLE
Backward compatibility with Xcode 13 for IOS

### LINKED ISSUE #1197: Allow user to use Xcode 13
As Realm JS does we should handle this gracefully. See https://github.com/realm/realm-js/pull/5531

### PR DESCRIPTION
**Pull request by desistefanova**
In relation to https://developer.apple.com/forums/thread/716965
Added flag `-fno-objc-msgsend-selector-stubs` for IOS
Fixes #1197

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -37,16 +38,19 @@ else()
...
 elseif(CMAKE_SYSTEM_NAME STREQUAL "Linux")
     string(APPEND OUTPUT_DIR "/linux")
 elseif(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
     string(APPEND OUTPUT_DIR "/macos")
+    add_cxx_flag_if_supported(-fno-objc-msgsend-selector-stubs)
 elseif(CMAKE_SYSTEM_NAME STREQUAL "Android")
     string(APPEND OUTPUT_DIR "/android/${CMAKE_ANDROID_ARCH_ABI}")
 elseif(CMAKE_SYSTEM_NAME STREQUAL "iOS")
     string(APPEND OUTPUT_DIR "/ios")
+    add_cxx_flag_if_supported(-fno-objc-msgsend-selector-stubs)
```
Comment (blagoev) [inline src/CMakeLists.txt:53]: I would add these later in the 
`elseif(CMAKE_SYSTEM_NAME STREQUAL iOS)`
and would add another branch for macOS as 
`elseif(CMAKE_SYSTEM_NAME STREQUAL "Darwin")`

**Review by blagoev — CHANGES_REQUESTED** [review]

File: src/CMakeLists.txt (hunk 1 above)
Comment (desistefanova) [reply src/CMakeLists.txt:53]: Ok, I will return it backwards.

**Review by blagoev — APPROVED** [review]
looks ok. 
lets do the testing before merging

**Comment by desistefanova** [comment]
We don't need this change, since the framework is dynamic, not static.

**Comment by desistefanova** [comment]
We don't need this change, since the framework is dynamic, not static.

**Pull request closed without merging**